app/src/main/java/edu/uc/group/rankine/dto/ElementObject.kt

```diff
@@ -5,7 +5,7 @@ package edu.uc.group.rankine.dto
  */
 data class ElementObject(var name: String)
 {
-
-
-
+    override fun toString(): String {
+        return name
+    }
```

Comment:
This is doing the same thing as the default toString, provided by Kotlin, so this does not add any value... I'd remove it.